homeassistant/core.py

```diff
@@ -432,7 +432,7 @@ def async_fire(self, event_type: str, event_data=None,
         """
         if event_type != EVENT_HOMEASSISTANT_STOP and \
                 self._hass.state == CoreState.stopping:
-            raise ShuttingDown('Home Assistant is shutting down.')
+            raise ShuttingDown("Home Assistant is shutting down")
```

Comment:
For Logger messages and some other natural language messages in a lot of cases we are using double quotes and no periods at the end. Constants are usually just wrapped with single quotes. As far as I remember doesn't PEP say something about this but when ever I came across a file I will update it to match the majority of your files to keep the style synced.

Here I was a bit too eager to align the usage of the quotes that I touched the exceptions.